ios/FluentUI/Tooltip/TooltipView.swift

```diff
@@ -213,7 +213,7 @@ class TooltipView: UIView {
                                           text: String,
                                           tokenSet: TooltipTokenSet,
                                           isMessage: Bool) -> CGSize {
-        let boundingWidth = min(tokenSet[.maximumWidth].float - (2 * TooltipTokenSet.paddingHorizontal), size.width)
+        let boundingWidth = UIApplication.shared.preferredContentSizeCategory.isAccessibilityCategory ? size.width : tokenSet[.maximumWidth].float - (2 * TooltipTokenSet.paddingHorizontal)
```

Comment:
we should be avoiding using UIApplication.Shared usuage @jeaniehuynh

Reply:
ah okay, what's the alternative? also curious as to why